doc/POSTAPOC_PRICE_GUIDE.md

```diff
@@ -99,7 +99,7 @@ pretty much the most anyone will be willing to spend on any one thing, no matter
 ```
 
 #### Food pricing
-Food is priced mostly for a combination of calories, how preservable they are, and how nutritious they are.
+Food is priced mostly for a combination of calories, how perservable they are, and how nutritious they are.
```

Comment:
"preservable" is right here, I suppose. It means "how long it can last while preserved" or something like that.

Reply:
That's confusing. I don't think "perservable" (the "correction") is a word. Maybe "their (extendable) shelf life" or something like that might be used.

Reply:
https://www.thefreedictionary.com/preservable

Reply:
I'm still confused. The reference is to pREservable as far as I can see (and what the link itself says), not the correction's pERservable.